haskell-src/Concordium/Types/Parameters.hs

```diff
@@ -158,14 +799,28 @@ instance HashableTo Hash.Hash TransactionFeeDistribution where
 
 instance Monad m => MHashableTo m Hash.Hash TransactionFeeDistribution
 
+-- * GAS rewards
+
+-- |Constraint on a type level 'GASRewardsVersion' that can be used to get a corresponding
+-- 'SGASRewardsVersion'. An alias for 'SingI'.
```

Comment:
"An alias for singI" does not provide value to the reader. That's clear from the definition.